doc/api/stream.md

```diff
@@ -1667,6 +1667,12 @@ of a stream that are intended for use by consumers (as described in the
 [API for Stream Consumers][] section). Doing so may lead to adverse side effects
 in application code consuming the stream.
 
+It is highly discouraged to override any public method or to emit
+`'error'` events manually through `.emit(err)` instead of using API-provided
```

Comment:
> We also do not have "internal" events, so it's even more difficult to prevent any such usage.

I think that @BridgeAR was referring to "stream events" not only `'errors'` but also `'data'`, `'finish'`, `'end'`, `'close'`, etc.

Reply:
I see. Not sure what to do with that though? Should I add "internal events" to the paragraph?